test/integration/goldens/library/LibraryServiceClient.java

```diff
@@ -404,6 +479,16 @@ public final Shelf mergeShelves(String name, ShelfName otherShelfName) {
    * <p>Returns NOT_FOUND if either shelf does not exist. This call is a no-op if the specified
    * shelves are the same.
    *
+   * <p>Sample code:
+   *
+   * <pre>{@code
+   * try (LibraryServiceClient libraryServiceClient = LibraryServiceClient.create()) {
+   *   String name = "name3373707";
+   *   String otherShelfName = "other_shelf_name145746959";
```

Comment:
FYI this will be fixed in #585 (lower_snake_case -> lowerCamelCase).

Reply:
Nice. Approved #585